src/main/java/org/schemaspy/SchemaAnalyzer.java

```diff
@@ -76,6 +71,9 @@ public SchemaAnalyzer(SqlService sqlService, DatabaseService databaseService, Co
         this.databaseService = Objects.requireNonNull(databaseService);
         this.commandLineArguments = Objects.requireNonNull(commandLineArguments);
         addOutputProducer(new XmlProducerUsingDOM());
+        if(commandLineArguments.isRenderDotInJvm()){
+            Dot.setInstance(commandLineArguments.isRenderDotInJvm(), commandLineArguments.getOutputDirectory().getName());
```

Comment:
Wonder if this could be put in the SchemaSpyConfiguration instead.
It would be a step in the right direction regarding dependency injection.
Possibly due to the current state instead of setInstance have a getOrCreateInstance() and have it return that instance. It would then be present in the SpringContext.

But it can be done later.